sql/testdata/select

```diff
@@ -120,3 +120,35 @@ SELECT *, "from", kw."from" FROM kw
 ----
 from from from
 1    1    1
+
+# SELECT from index.
+
+statement ok
+CREATE TABLE xyz (
+  x INT PRIMARY KEY,
+  y INT,
+  z INT,
+  CONSTRAINT foo INDEX (z, y)
+)
+
+statement ok
+INSERT INTO xyz VALUES (4, 5, 6), (1, 2, 3);
+
+query II
+SELECT * FROM xyz@foo
+----
+3 2
+6 5
+
+query I
+SELECT z FROM test.xyz@foo WHERE y = 5
+----
+6
+
+query error table "unknown" does not exist
+SELECT z FROM test.unknown@foo WHERE y = 5
```

Comment:
Yes, though really I just misunderstood your comment and missed that error case. Good thing to add it because it found a bug.